packages/development/src/components/customPlugins/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,66 @@
+import { generateId, SlateElement, YooptaBlockData } from '@yoopta/editor';
```

Comment:
I have a very strong feeling that `example` should be stored within plugin and be generated to the website by Lerna, e.g.

in your plugin you have:
```
./README.md - docs
./test - tests folder
./example/
```

Also it gives a bit more understanding to the person who looks into the plugin repo of how to use it.